src/common_types.ts

```diff
@@ -38,29 +38,26 @@ export type StatusValue = 'succeeded' | 'failed' | 'skipped';
 export type Reporters = keyof typeof reporters;
 
 export type TraceOutput = {
-  name?: string;
+  name: string;
   start: number;
+  type: string;
   end?: number;
 };
 
-export type UserTiming = TraceOutput & {
-  name: string;
-  type: string;
-  duration?: number;
+export type PerfMetrics = {
+  fcp: number;
```

Comment:
I thought about that, but keeping it sync with RUM agent intake `transaction.experience.fcp` vs `browser.experience.fcp`. Thought it would be cleaner in the UI.

Reply:
I think that now would be an OK time to add the units in. Maybe this is a case where synthetics leads and RUM follows? Elsewhere in synthetics we use `.us` so it's a question of where we keep consistency.

Reply:
I'm used to having a `*.us` field from beats; there isn't going to be any annotation when I'm looking at documents in an Elasticsearch response, so that explicitness leaves no doubt.

Reply:
I think it's always helpful to have units